PantheonCMD/pcentrypointvalidator.py

```diff
@@ -0,0 +1,126 @@
+#!/usr/bin/python3
+
+import argparse
+import re
+import os
+from pcchecks import Regex
+import sys
+from pcutil import get_exist, get_not_exist
+from pcprvalidator import get_no_prefix_files, get_all_modules, get_all_assemblies, get_undetermined_files
+from pcvalidator import validation
+from pcmsg import print_message, print_report_message
+
+parser = argparse.ArgumentParser()
+
+
+def get_nonexisting_entry_points(entry_point_list):
+    nonexistent_files = get_not_exist(entry_point_list)
+
+    if nonexistent_files:
+        print_message(nonexistent_files, 'entry point', 'does not exist in your repository')
+        sys.exit(2)
+
+
+def get_includes(entry_points):
```

Comment:
You might run into a similar problem I encountered with the coalescing function here.

Namely, some users use attributes in their includes - and nested ones at that. As such, unless you go through and resolve the entire attribute tree, attempting to convert an include into an actual path sometimes fails.

How would you feel about a pared down version of the coalescing loop to still work with attributes and iterate through levels of content?